deploy/rp-production-parameters.json

```diff
@@ -2,6 +2,9 @@
     "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
     "contentVersion": "1.0.0.0",
     "parameters": {
+        "AzureSecPackVSATenantId": {
```

Comment:
This should be lowercase to be consistent with the other params and to match the json from the "config".